scripts/murmur.ini

```diff
@@ -133,7 +133,7 @@ serverpassword=
 
 ; Maximum bandwidth (in bits per second) clients are allowed
 ; to send speech at.
-bandwidth=72000
+bandwidth=558000
```

Comment:
I am not sure we actually want to default to this value. This could cause inexperienced server admins to just leave this setting at the default and then wondering why their server has bandwidth problems after a hand full of clients are connected (that all make use of this) :thinking:

Reply:
Considering how situational that is it would be a problem with any value. That is to say, 72000 could be an issue for people with low bandwidth or many clients sharing rooms.

A common issue with 72000 is that settings in the GUI configuration often exceed it.

Reply:
Fair point :+1: